flink-core/src/main/java/org/apache/flink/util/TimeUtils.java

```diff
@@ -29,9 +33,23 @@
  */
 public class TimeUtils {
 
+	private static final Map<String, ChronoUnit> LABEL_TO_UNIT_MAP = Collections.unmodifiableMap(initMap());
+
 	/**
 	 * Parse the given string to a java {@link Duration}.
-	 * The string is like "123ms", "321s", "12min" and such.
+	 * The string is in format "{length value}{time unit label}", e.g. "123ms", "321 s".
+	 * If no time unit label is specified, it will be considered as milliseconds.
+	 *
+	 * <p>Supported time unit labels are:
```

Comment:
Is there a source/standard support this format? If so, could you refer it?

Reply:
The supported labels are acquired from the source code of [scala.concurrent.duration.Duration](https://github.com/scala/scala/blob/f7b5b312ca5dd1333245d8174a8dbadc89bd40fc/src/library/scala/concurrent/duration/Duration.scala#L76). 
And I think there is no need to refer it, as we want to be compatible with scala duration labels at the moment but not with scala duration's future changes.

The standard for users would only be the javadoc of flink TimeUtils then.

Reply:
Make sense. "refer it" means we keep the context at least in this pull request so that the origin of the change can be sourced. I just notice that it is already mentioned in the title and description :P